question28/js/forgotpassword.js

```diff
@@ -0,0 +1,30 @@
+import { validateInputValue } from "./validations";
+import { Chance } from "chance";
+const chance = new Chance();
+
+const submitButton = document.getElementById('js-submit-button');
+const userInfoInputArea = document.getElementById('mail');
+const mailInputArea = document.getElementById('mail');
```

Comment:
you are getting same from same Id. you should get only once

Reply:
Thank you for your review! You are right...I completely missed it. Fixed so please confirm.

fixed commit : https://github.com/csakyo/homework/pull/42/commits/86b2b0651ce62a9e6830dc64dd53f76f313c63e5